core/tests/ledger_cleanup.rs

```diff
@@ -351,15 +351,14 @@ mod tests {
             &ledger_path,
             if config.fifo_compaction {
                 BlockstoreOptions {
-                    column_options: LedgerColumnOptions {
-                        shred_storage_type: ShredStorageType::RocksFifo(
-                            BlockstoreRocksFifoOptions {
-                                shred_data_cf_size: config.shred_data_cf_size,
-                                ..BlockstoreRocksFifoOptions::default()
-                            },
-                        ),
-                        ..LedgerColumnOptions::default()
-                    },
+                    column_options: LedgerColumnOptions::new(
+                        ShredStorageType::RocksFifo(BlockstoreRocksFifoOptions {
+                            shred_data_cf_size: config.shred_data_cf_size,
+                            ..BlockstoreRocksFifoOptions::default()
+                        }),
+                        BlockstoreCompressionType::default(),
+                        0,
```

Comment:
For the next person reading this code, please add a comment what this zero is; as in
```rust
  0, // rocks_perf_sample_interval
```